### PR TITLE
Feature/15 add a sign in form

### PR DESCRIPTION
**Pull request by cristicatalan**
Trello card:
https://trello.com/c/w3tJ6un4/15-add-a-sign-in-form

### CODE REVIEW

File: lib/dry_twitter/views/sign_in.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    end
+
+    def locals(options = {})
+      if options.size > 0
+        super.merge(
+            user: options[:input]["user"],
+            errors: options[:input][:errors]
+        )
+      else
+        super.merge(user: nil, errors: nil)
```
Comment (alexandru-calinoiu) [inline lib/dry_twitter/views/sign_in.rb]: We can use a null guard operator to make this nicer.

File: lib/dry_twitter/views/sign_in.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      configure do |config|
+        config.template = "sign_in"
+      end
+    end
+
+    def locals(options = {})
+      if options.size > 0
+        super.merge(
+            user: options[:input]["user"],
+            errors: options[:input][:errors]
```
Comment (alexandru-calinoiu) [inline lib/dry_twitter/views/sign_in.rb]: Mixing string and symbol guys is a great way to create confusion.

File: lib/dry_twitter/views/sign_in.rb (hunk 1 above)
Comment (alexandru-calinoiu) [reply lib/dry_twitter/views/sign_in.rb]: TIP: look at the [dig](https://ruby-doc.org/core-2.5.0/Hash.html#method-i-dig) method.

**Review by alexandru-calinoiu — CHANGES_REQUESTED** [review]
It can be done better.

File: lib/dry_twitter/views/sign_in.rb (hunk 2 above)
Comment (cristicatalan) [reply lib/dry_twitter/views/sign_in.rb]: I'm aiming to use symbols whenever they can be used. In this instance the stringed key is coming from the framework.

**Comment by alexandru-calinoiu** [comment]
Closing this as is added in https://github.com/Agilefreaks/dry-twitter/pull/13

**Pull request closed without merging**